.github/workflows/codeql-analysis.yml

```diff
@@ -5,10 +5,16 @@ on:
     branches:
       - master
       - branch/*
+    paths-ignore:
+      - 'docs/**'
```

Comment:
You can probably also ignore

- assets
- build.assets
- docker
- dronegen
- examples
- vagrant

Reply:
I see `*.go` files in some of those. Seems like we'd want to scan them, but I'm guessing the default `make` doesn't build them? Probably could set up separate jobs for them so they get scanned.